WordPress/src/main/java/org/wordpress/android/ui/stockmedia/StockMediaPickerActivity.java

```diff
@@ -395,12 +398,18 @@ public void onStockMediaUploaded(MediaStore.OnStockMediaUploaded event) {
             ToastUtils.showToast(this, R.string.media_upload_error);
             AppLog.e(AppLog.T.MEDIA, "An error occurred while uploading stock media");
         } else {
-            ArrayList<Integer> idList = new ArrayList<>();
-            for (MediaModel media : event.mediaList) {
-                idList.add(media.getId());
+            int count = event.mediaList.size();
+            long[] idArray = new long[count];
+            for (int i = 0; i < count; i++) {
+                idArray[i] = event.mediaList.get(i).getMediaId();
             }
+
+            Map<String, Integer> properties = new HashMap<>();
+            properties.put("count", count);
+            AnalyticsTracker.track(AnalyticsTracker.Stat.STOCK_MEDIA_UPLOADED, properties);
```

Comment:
I wonder if we should reply here the approach followed when bumping analytics for other photos. 
Analytics are bumped for each picture on its own call, by adding properties of the media to the event. See `trackAddRecentMediaEvent` in `PhotoPickerFragment` and how we used `AnalyticsUtils.getMediaProperties` to attach properties of the media to the event.